### PR TITLE
WX-1627 Belated addition of v87 changelog entries

### PR DESCRIPTION
**Pull request by jgainerdewar**
[Link to rendered markdown](https://github.com/broadinstitute/cromwell/blob/352c7ee278911d37074719ab906ec4c10eac362e/CHANGELOG.md)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,13 +13,33 @@ The `genomics` configuration entry was renamed to `batch`, see [ReadTheDocs](htt
 
 ## 87 Release Notes
 
+### GCP Batch
+
+ * Added NVidia driver install ([#7235](https://github.com/broadinstitute/cromwell/pull/7235}))
```
Comment (aednichols) [inline CHANGELOG.md:18]:
```suggestion
 * Added Nvidia driver install (default 418) ([#7235](https://github.com/broadinstitute/cromwell/pull/7235}))
```
The driver version matters for what hardware it supports.

File: CHANGELOG.md [hunk 2]
```diff
@@ -36,6 +56,17 @@ In a future release, **delocalization** will also migrate to `gcloud storage`. A
 
 [3] Cromwell itself uses crc32c hashes for call caching and is not affected
 
+
+### Other Improvements
+ * In certain cases DRS downloads have been found to hang forever Cromwell will now time these out ([#7416](https://github.com/broadinstitute/cromwell/pull/7416))
```
Comment (aednichols) [inline CHANGELOG.md:61]:
```suggestion
 * In certain cases DRS downloads have been found to hang forever. Cromwell will now time these out. ([#7416](https://github.com/broadinstitute/cromwell/pull/7416))
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -36,6 +56,17 @@ In a future release, **delocalization** will also migrate to `gcloud storage`. A
...
+ * Increased default Akka `client.parsing.max-response-reason-length` to 1024 ([#7406](https://github.com/broadinstitute/cromwell/pull/7406))
+ * Workflow Completion Callback bodies now include fully-qualified output names ([#7234](https://github.com/broadinstitute/cromwell/pull/7234))
+ * Improved workflow abort error handling ([#7245](https://github.com/broadinstitute/cromwell/pull/7245))
+ * Improved logging for troubleshooting ([#7246](https://github.com/broadinstitute/cromwell/pull/7246)) ([#7253](https://github.com/broadinstitute/cromwell/pull/7253)) ([#7388](https://github.com/broadinstitute/cromwell/pull/7388))
+ * Support for Intel Ice Lake chips in PAPI backend ([#7252](https://github.com/broadinstitute/cromwell/pull/7252))
```
Comment (aednichols) [inline CHANGELOG.md:66]:
```suggestion
 * Support for Intel Ice Lake chips in Life Sciences backend ([#7252](https://github.com/broadinstitute/cromwell/pull/7252))
```
Lingo leakage – people who started with Cromwell in the last few years don't know we call all versions of  the API "PAPI"

**Review by aednichols — APPROVED** [review]

**Review by rsaperst — COMMENTED** [review]
Would it be a good idea to mention #7414 since it introduced a configurable field to users? Maybe something like "Added configuration option to remove 4 byte UTF-8 encoded characters (e.g. emojis) from metadata strings"?

**Comment by jgainerdewar** [comment]
@rsaperst Excellent call-out, I did not include that here because the important bug-fix to that feature did not make it in to v87.

**Review by rsaperst — APPROVED** [review]
@jgainerdewar Got it, in that case this looks good to me!

**Review by aednichols — APPROVED** [review]

**Review by THWiseman — APPROVED** [review]

**Pull request merged**